runtime/compiler/control/CompilationThread.cpp

```diff
@@ -2033,7 +2033,7 @@ bool TR::CompilationInfo::shouldAbortCompilation(TR_MethodToBeCompiled *entry, T
 
    if (entry->_unloadedMethod) // method was unloaded while we were trying to compile it
       {
-      TR_ASSERT(entry->_compErrCode == compilationInterrupted, "if method was unloaded we must return we an error code of compilationInterrupted");
+      TR_ASSERT(entry->_compErrCode == compilationInterrupted, "Recevied error code %u, expect compilationInterrupted when the method was unloaded", entry->_compErrCode);
```

Comment:
Small typo: Recevied -> Received

Reply:
My bad. Updated and ready for review. Thanks!